templates/mrepo.conf.erb

```diff
@@ -18,6 +18,12 @@ rhnlogin = <%= @rhn_username %>:<%= @rhn_password %>
 rhnlogin = <%= @rhn_username %>
 <% end -%>
 
-<% unless (mailto = @mailto) == 'UNSET' -%>
-mailto = <%= mailto %>
+<% if @mailto -%>
+mailto = <%= @mailto %>
+<%   if @mailfrom -%>
```

Comment:
Ah I see. Yeah seems to be a matter of taste, but both make sense 👍 

http://stackoverflow.com/questions/9746035/whats-the-best-practice-when-indenting-erb-control-structures-in-html-markup